catalog/transformer/catalog-transformer-streaming-impl/src/test/java/org/codice/ddf/transformer/xml/streaming/impl/Gml3ToWktImplTest.java

```diff
@@ -83,4 +92,92 @@ public void testConvertEnvelope() throws ValidationException {
     String wkt = gtw.convert(gml);
     assertThat(wkt, equalTo("POLYGON ((0 0, 0 1, 1 1, 1 0, 0 0))"));
   }
+
+  @Test
+  public void testGmlPointToWktValidSrsName() throws ValidationException {
+    assertThat(gtw.convert(createSrsAwareGmlPoint(0, 0, "EPSG:4326")), is(createWktPoint(0, 0)));
+    assertThat(gtw.convert(createSrsAwareGmlPoint(0, 1, "EPSG:4326")), is(createWktPoint(0, 1)));
+    assertThat(gtw.convert(createSrsAwareGmlPoint(1, 0, "EPSG:4326")), is(createWktPoint(1, 0)));
+    assertThat(gtw.convert(createSrsAwareGmlPoint(1, 1, "EPSG:4326")), is(createWktPoint(1, 1)));
+  }
+
+  @Test
+  public void testGmlPointToWktValidSrsNameUrn() throws ValidationException {
+    assertThat(
+        gtw.convert(createSrsAwareGmlPoint(0, 0, "urn:x-ogc:def:crs:EPSG:4326")),
+        is(createWktPoint(0, 0)));
+    assertThat(
+        gtw.convert(createSrsAwareGmlPoint(0, 1, "urn:x-ogc:def:crs:EPSG:4326")),
+        is(createWktPoint(0, 1)));
+    assertThat(
+        gtw.convert(createSrsAwareGmlPoint(1, 0, "urn:x-ogc:def:crs:EPSG:4326")),
+        is(createWktPoint(1, 0)));
+    assertThat(
+        gtw.convert(createSrsAwareGmlPoint(1, 1, "urn:x-ogc:def:crs:EPSG:4326")),
+        is(createWktPoint(1, 1)));
+  }
+
+  @Test
+  public void testGmlPointToWktValidSrsNameUrl() throws ValidationException {
+    assertThat(
+        gtw.convert(createSrsAwareGmlPoint(0, 0, "http://www.opengis.net/def/crs/EPSG/0/4326")),
```

Comment:
❓ It looks like the "0" in the 4326 URI is a version number.  Do other versions work?
```
http://www.opengis.net/def/crs/EPSG/8.5/4326
http://www.opengis.net/def/crs/EPSG/9.5/4326
```